### DISCUSSION TITLE
What are the downsides of increasing quota-backend-bytes?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [etcd-io/etcd#14372] [Q&A]
What are the downsides of increasing quota-backend-bytes?

**Comment by SimFG** [comment]
In order to ensure the normal operation of the etcd service, more memory is required if increasing quota-backend-bytes, and maybe degrade performance.

**Comment by martinenkoEduard** [comment]
Does quota-backend-bytes means more HDD space? Or RAM also?

**Reply by SimFG** [reply]
The larger quota-backend-bytes means more `kv` can be stored, which need more HDD space. And When more and more kvs are stored, more RAM is required.
